website/docs/r/service_level.html.markdown

```diff
@@ -73,21 +73,21 @@ All nested `events` blocks support the following common arguments:
     * `where` - (Optional) A filter that specifies all the NRDB events that are considered in this SLI (e.g, those that refer to a particular entity).
     * `select` - (Optional) The NRQL SELECT clause to aggregate events.
       * `attribute` - (Optional) The event attribute to use in the SELECT clause.
-      * `function` - (Required) The function to use in the SELECT clause. Valid values are `COUNT`and `SUM`.
+      * `function` - (Required) The function to use in the SELECT clause. Valid values are `COUNT` and `SUM`.
```

Comment:
Based on the [validation](https://github.com/newrelic/terraform-provider-newrelic/pull/2293/files#diff-8fd5027c85dd86ebb1077c41695d8bde99ec1108b9d7120796f8d8a5e53a3513R154) in the schema, it looks like we need to add the new valid values here, correct?